### PR TITLE
Docsp 8054 run a command usage

### PR DESCRIPTION
**Pull request by mongomoe**
Stage:
------
https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-8054-Run-a-Command-Usage/usage-examples/command

JIRA:
-----
https://jira.mongodb.org/browse/DOCSP-8054

### CODE REVIEW

File: source/usage-examples/command.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
+=============
+Run A Command
+=============
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can run all :manual:`non CRUD </crud/>` database operations using the
```
Comment (cbush) [inline source/usage-examples/command.txt]: "non CRUD" in the link feels off, since there should be a hyphen, and the link leads to CRUD, not a definition of "non-CRUD". I'd suggest rewording it to avoid the "non-" part.

File: source/usage-examples/command.txt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+--------
+
+You can run all :manual:`non CRUD </crud/>` database operations using the 
+:node-api:`command() </Admin.html#~command>` method. Typically ``command()`` is used for diagnostic and administrative tasks, such as fetching server stats or initializing a replica set. 
+
+.. note::
+    Using :manual:`collection methods </reference/method/js-collection>`
+    instead of raw database commands through the ``command()`` is
+    recommended for beginner users, since ``collection methods`` are
+    wrappers that abstract some configuration.
```
Comment (cbush) [inline source/usage-examples/command.txt]: Can you clarify this? I'm not sure this is coming across so clearly.

File: source/usage-examples/command.txt (hunk 1 above)
Comment (mongomoe) [reply source/usage-examples/command.txt]: updated to "You can run all raw database operations, not including CRUD operations, using the command() method."

File: source/usage-examples/command.txt (hunk 2 above)
Comment (mongomoe) [reply source/usage-examples/command.txt]: updated to "Use collection methods instead of raw db commands whenever possible."
